### ISSUE TITLE
[Scenario] Não Olhe Para Trás

### ISSUE THREAD

**Issue by Diesdooi** [NPBruce/valkyrie#1507]
### What kind of mission do you want to add (mark with an X, e.g. [X] Descent)?
- [X] Mansions of Madness
- [ ] Descent

### What expansions does the scenario require?
- [ ] Conversion Kit (CK)
- [ ] First Edition (MoM1E) - Also Recurring Nightmares
- [ ] Fatal Alchemy (FA) - Also Suppressed Memories
- [ ] Call of the Wild (CotW) - Also Suppressed Memories
- [ ] Beyond the Threshold (BtT)
- [ ] Streets of Arkham (SoA)
- [ ] Sunctum of Twilight (SoT)
- [ ] Horrific Journeys (HJ)
- [X] Path of the Serpent (PotS)

### What expansions can the scenario optionally use?
- [ ] Conversion Kit (CK)
- [ ] First Edition (MoM1E) - Also Recurring Nightmares
- [ ] Fatal Alchemy (FA) - Also Suppressed Memories
- [ ] Call of the Wild (CotW) - Also Suppressed Memories
- [X] Beyond the Threshold (BtT)
- [X] Streets of Arkham (SoA)
- [X] Sunctum of Twilight (SoT)
- [X] Horrific Journeys (HJ)
- [ ] Path of the Serpent (PotS)

https://github.com/Diesdooi/naoolheparatras

**Comment by antontimmermans** [comment]
added scenario o manifest and statistics overvews. Issue can be closed

**Comment by scrubbless** [comment]
Hi @antontimmermans 

Thanks for adding this. 
I noticed the grabber was throwing an error, looks like you spelt the .ini name wrong: 
![image](https://user-images.githubusercontent.com/32058089/123799942-a5bf8c80-d8e0-11eb-8b50-0696efa1a770.png)
I've done this many a time too!

I've corrected the manifest, just as a tip. I usually paste the full RAW path in after the "external=" and then cut the "name.ini" off the end, paste is into the [ ] and then delete off the .ini. That way I can be sure I don't type the ini file name wrong.
